diff --git a/node_modules/mulmocast/lib/utils/context.js b/node_modules/mulmocast/lib/utils/context.js
index 7cbb8ad..7f00f9e 100644
--- a/node_modules/mulmocast/lib/utils/context.js
+++ b/node_modules/mulmocast/lib/utils/context.js
@@ -12,7 +12,7 @@ const mulmoCredit = (speaker) => {
             type: "image",
             source: {
                 kind: "url",
-                url: "https://github.com/receptron/mulmocast-cli/raw/refs/heads/main/assets/images/mulmocast_credit.png",
+                url: "https://fordoanalysis-edit.s3.ap-northeast-1.amazonaws.com/assets/images/mulmo_credit.png",
             },
         },
         audio: {
